quickstart-examples/sagemaker-example/README.md

```diff
@@ -0,0 +1,122 @@
+# Iris Flower Classification Example - Steps to Launch Solution
+
+## Prerequisites
+
+- Docker up and running
+- AWS CLI
+- Python 3
+
+## Upload Model Artifacts to S3
+
+**Note**: Update `profile_name` and `region_name` in `sagemaker-example/iris_classifier/upload.py` if you are not using AWS `default` profile.
+
+```bash
+$ cd sagemaker-example/iris_classifier
+$ python3 upload.py
+
+# Output
+ModelDataS3URI: s3://sagemaker-ap-south-1-166187639221/model.tar.gz
+ModelECRImageURI: 720646828776.dkr.ecr.ap-south-1.amazonaws.com/sagemaker-scikit-learn:0.23-1-cpu-py3
```

Comment:
This is a private image. You can remove this.

Reply:
Agreed. Replaced AWS account ID with some random ID.